### PR TITLE
[AVFoundation] Update bindings for Xcode 8.2

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

File: src/avfoundation.cs [hunk 1]
```diff
@@ -2573,6 +2573,9 @@ interface AVAsset : NSCopying {
 		NSString WasDefragmentedNotification { get; }
 #endif
 
+		[iOS (10, 3), Mac (10,12), TV (10, 3)]
```
Comment (dalexsoto) [inline src/avfoundation.cs]: iOS 10,3? I think you mean 10,2? Also for mac maybe it is 10,12,2

Comment (mandel-macaque) [inline src/avfoundation.cs]: Agh, true, typo

File: src/avfoundation.cs [hunk 2]
```diff
@@ -2573,7 +2573,7 @@ interface AVAsset : NSCopying {
 		NSString WasDefragmentedNotification { get; }
 #endif
 
-		[iOS (10, 3), Mac (10,12), TV (10, 3)]
+		[iOS (10, 2), Mac (10,12,2), TV (10, 3)]
```
Comment (dalexsoto) [inline src/avfoundation.cs]: Not sure about TV 10,3 either ;)

File: src/avfoundation.cs [hunk 3]
```diff
@@ -2573,6 +2573,9 @@ interface AVAsset : NSCopying {
 		NSString WasDefragmentedNotification { get; }
 #endif
 
+		[iOS (10, 2), Mac (10,12,2), TV (10, 2)]
```
Comment (dalexsoto) [inline src/avfoundation.cs]: I hate to be that guy but I think you want TV (10,1) look at the Beta versions listed on https://developer.apple.com/download/

Comment (mandel-macaque) [inline src/avfoundation.cs]: No worries, but in the diff I see NS_AVAILABLE(10_12_3, 10_3) and in the Apple docs the supported platforms are:
* iOS 10.3+
* macOS 10.12.3+
* tvOS 10.3+

So I'm more inclined in changing the Mac attribute than the TV one, what do you think?

**Comment by dalexsoto** [comment]
build

**Comment by dalexsoto** [comment]
Test failure unrelated to this PR

**Pull request merged**